api/illumio/cloud/k8sclustersync/v1/k8s_info.proto

```diff
@@ -353,6 +378,7 @@ message LogEntry {
 // Message to encapsulate a request to send logs.
 message SendLogsRequest {
   oneof request {
+    Keepalive keepalive = 2;
```

Comment:
I don't understand why you'd need to partially pull changes from unmerged PR #155.
Revert this.
```suggestion
```